### ISSUE TITLE
Issue in Sample PhunMaps 

### ISSUE THREAD

**Issue by MuraliNallusami** [phunware/maas-mapping-ios-sdk#69]
Install the pod and fix the compatible issue. 

Once you switch map and directory. UI is blocking for few seconds. I Guess SDK which is supposed to run on thread is running in main thread. Can you please let me know, is this SDK issue?

**Comment by pdunshee-phunware** [comment]
Yes we have identified an issue with the SDK and will work to fix it, thank you for letting us know what you saw! In the meantime, we have also pushed an update to PhunMaps to work around this by using the provided imageURL of the POI and an image caching library. We will make the image caching internal to the SDK in a future update which should fix this issue. Will the sample app update be enough to get you unblocked for now?

**Comment by ghost** [comment]
Closing stale issues - if this persists, please let us know by opening a new issue. Thanks!
